### ISSUE TITLE
Problem with link to datasets

### ISSUE THREAD

**Issue by alefiury** [Computer-Vision-in-the-Wild/Elevater_Toolkit_IC#11]
Currently I am having the following error: 

requests.exceptions.HTTPError: 409 Client Error: Public access is not permitted on this storage account. for url: https://cvinthewildeus.blob.core.windows.net/datasets/classification/cifar_10_20211007/test.txt

It seems like we do not have access to the datasets anymore, can anyone check on this?

**Comment by NebiyouTen** [comment]
Yeah I am getting the same 409 access denied error. Did you manage to find a solution?

**Comment by 1338199** [comment]
Having the same problem.

**Comment by ChunyuanLI** [comment]
Note:  The data is on Azure Storage Blob, a SAS with Read permission is provided. Please append the following SAS at the end of each link to download: 
```bash
?sp=r&st=2023-08-28T01:41:20Z&se=3023-08-28T09:41:20Z&sv=2022-11-02&sr=c&sig=Msoq5dIl%2Fve6F01edGr8jgcZUt7rtsuJ896xvstSNfM%3D
```

**Comment by haotian-liu** [comment]
Hi, the toolkit has been updated with the latest SAS and it should be able to download automatically now.

**Comment by pierowu** [comment]
It seems that the link still cannot be accessed. I don't know whether it's my internet problem or not. Would you mind share a copy by disk or something?
![image](https://github.com/Computer-Vision-in-the-Wild/Elevater_Toolkit_IC/assets/61963313/313b1f78-eb6a-4e05-aaec-4d4f793cb835)

**Comment by haotian-liu** [comment]
Please check these two files to ensure the new blob SAS is included (you may need to pull the latest code base)

https://github.com/Computer-Vision-in-the-Wild/Elevater_Toolkit_IC/blob/main/vision_benchmark/common/constants.py#L4
https://github.com/Computer-Vision-in-the-Wild/Elevater_Toolkit_IC/blob/main/vision_benchmark/evaluation/feature.py#L561

And please let us know if the issue persists, thanks.

**Comment by pierowu** [comment]
After updating links in the above two files, now the datasets can be download successfully.
Thank you for helping!
